tenable/io/v3/was/user_templates/schema.py

```diff
@@ -0,0 +1,27 @@
+from marshmallow import Schema, fields, validate
```

Comment:
Please Add the docstring here.
for example,
'''
User Template API Endpoint Schemas
'''